app/components/minigames/Memory/MemoryPage.tsx

```diff
@@ -0,0 +1,17 @@
+import MemoryGame from "@/app/components/minigames/Memory/MemoryGame";
+import React from "react";
+
+const MemoryPage: React.FC = () => {
+    return (
+        <div className="flex flex-col justify-center items-center min-h-screen bg-star p-4">
+            <div className="text-center mb-4">
+                <h1 className="text-3xl font-bold mb-16 h1">Venus Entdeckungen: Ein Gedächtnisspiel</h1>
```

Comment:
The CSS class "text-3xl font bold h1" is unnecessary as the HTML-Element h1 is already defined in our global.css